.gitignore

```diff
@@ -22,3 +22,6 @@ vendor/
 tools/windows-node-installer/wni
 windows-node-installer.json
 winworker*
+
+# bindata files
+pkg/bootstrapper/bindata.go
```

Comment:
Why are we ignoring the bindata files? Once we run `go bin-data`, this file will be updated too.

Reply:
According to conversation [here](https://github.com/openshift/windows-machine-config-bootstrapper/pull/216#discussion_r447305876), `bindata.go` is generated as part of the build. Will exposing the `bindata` file be useful ?